.github/workflows/scheduled.yml

```diff
@@ -657,6 +664,30 @@ jobs:
             path: |
               /tmp/exchange_fuzzer_repro
 
+  row-number-fuzzer-run:
+    name: RowNumber Fuzzer
+    runs-on: ubuntu-latest
+    container: ghcr.io/facebookincubator/velox-dev:centos8
+    needs: compile
+    timeout-minutes: 120
+    steps:
```

Comment:
are there no artifacts to upload in case of failure here ?

Reply:
This fuzzer doesn't have the functionality of saving artifacts on failures yet.